components/InlineFeedbackPopover/Comment.tsx

```diff
@@ -54,6 +62,40 @@ const Comment: React.FC<CommentProps> = ({ comment, canEdit, onUpdateComment })
     })
   }
 
+  const [createCommentThanks] = useCreateCommentThanksMutation({
+    onCompleted: () => {
+      // just refetches the post as in updateComment
+      onUpdateComment()
+    },
+  })
+
+  const createNewCommentThanks = () => {
+    createCommentThanks({
+      variables: {
+        commentId: comment.id,
+      },
+    })
+  }
```

Comment:
This is just a personal style preference, so feel free to ignore it, but I prefer this kind of style:

```suggestion
  const [createCommentThanks] = useCreateCommentThanksMutation()

  const createNewCommentThanks =  async () => {
    await createCommentThanks({
      variables: {
        commentId: comment.id,
      },
    })
    await  onUpdateComment()
  }
```

because it keeps the logic together, as opposed to `onCompleted` being separated from where the thing calling it is.

Reply:
I agree! Great suggestion